src/lib/normalizeVNode.js

```diff
@@ -1,3 +1,29 @@
+import { isRenderableVNode, isTextNode } from "../utils/domUtils";
+
 export function normalizeVNode(vNode) {
-  return vNode;
+  if (!isRenderableVNode(vNode)) {
+    return "";
+  }
+
+  if (isTextNode(vNode)) {
+    return String(vNode);
+  }
+
+  if (typeof vNode.type === "function") {
+    const newVNode = normalizeVNode(
+      vNode.type({ ...vNode.props, children: vNode.children }),
+    );
+    return newVNode;
+  }
+
+  if (Array.isArray(vNode)) {
+    const vNodoes = vNode.map(normalizeVNode);
+    return vNodoes;
```

Comment:
오타인 것 같은데 `vNodes`로 수정하면 좋을 것 같습니다~
```suggestion
    const vNodes = vNode.map(normalizeVNode);
    return vNodes;
```

Reply:
오 감사합니다! 급하게 하느라 오타가 많이 심각하네요..